src/public/App.jsx

```diff
@@ -64,8 +65,22 @@ class App extends Component {
     }
   }
 
+  componentWillReceiveProps (nextProps) {
+    const { album, photos } = nextProps
+    if (photos.fetchStatus === 'failed' && album === null) {
+      this.setState({error: 'Fetch error'})
+    }
+  }
+
   render () {
     const { t, album, photos } = this.props
+    if (this.state.error) {
+      return (
+        <div className={classNames(styles['pho-public-layout'], styles['pho-public-layout--full'])}>
+          <ErrorShare errorType={`public_album_unshared`} />
```

Comment:
A `fetchStatus === 'failed'` could have other causes but this one.

In `componentWillReceiveProps` you should add test like in this PR https://github.com/cozy/cozy-drive/pull/336/files#diff-77a0342c324df1795ff84f7011d10e78R32